packages/forklift-console-plugin/src/modules/NetworkMaps/EmptyStateNetworkMaps.tsx

```diff
@@ -44,6 +46,20 @@ const EmptyStatePlans: React.FC<{ namespace: string }> = ({ namespace }) => {
               </Trans>
             </FlexItem>
           </Flex>
+        ) : (
+          <Trans t={t} ns="plugin__forklift-console-plugin">
+            Migration networks maps are used to map network interfaces between source and target
+            workloads
+          </Trans>
+        )
+      }
+      callForActionButtons={
+        hasSufficientProviders && (
+          <AddMappingButton
+            namespace={namespace}
+            mappingType={MappingType.Network}
+            label={t('Create NetworkMap')}
+          />
```

Comment:
This should match `NetworkMappingsPage`...

https://github.com/kubev2v/forklift-console-plugin/blob/c90e1513a4ee4d19c75992e4d7a4bde49b1f9ae0/packages/forklift-console-plugin/src/modules/NetworkMaps/NetworkMappingsPage.tsx#L94-L97

and be:

```suggestion
        hasSufficientProviders && <AddNetworkMappingButton namespace={namespace} />
```